.gitignore

```diff
@@ -2,6 +2,7 @@ composer.phar
 vendor/
 bin/
 php-git-hooks.yml
+.idea
```

Comment:
put this in your global gitignore.
environment specific folders and files should not be in project gitignore